tests/hazmat/backends/test_openssl.py

```diff
@@ -456,6 +458,32 @@ def test_unsupported_oaep_label_decrypt(self):
                 )
             )
 
+    def test_invalid_oaep_decryption(self):
```

Comment:
Yes, any future backend should also behave correctly here.

On Mon, Dec 28, 2015 at 12:08 AM, Paul Kehrer <notifications@github.com>
wrote:

> In tests/hazmat/backends/test_openssl.py
> <https://github.com/pyca/cryptography/pull/2599#discussion_r48463229>:
>
> > @@ -456,6 +458,32 @@ def test_unsupported_oaep_label_decrypt(self):
> >                  )
> >              )
> >
> > +    def test_invalid_oaep_decryption(self):
>
> Arguably this test exists to trigger a specific msg on the error stack in
> the openssl backend, but it definitely should raise the proper ValueError
> no matter the backend. We can move it (although I'd keep an altered version
> of the comment so we understand its value in the future) if you'd like.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/2599/files#r48463229>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084